### PR TITLE
community: add coze chat model

### PR DESCRIPTION
**Pull request by chyroc**
add coze chat model, to call coze.com apis

### CODE REVIEW

**Comment by chyroc** [comment]
@baskaryan please review this pr, thanks

**Review by baskaryan — COMMENTED** [review]
could we add a notebook to the docs?

**Comment by chyroc** [comment]
@baskaryan already add notebook docs.

and there are 2 issues, i dont understand which this mean.

first, https://github.com/langchain-ai/langchain/actions/runs/8842400719/job/24281000395?pr=20770, it say should import from langchain_community, i already do , i dont understand.

second, https://github.com/langchain-ai/langchain/actions/runs/8842400719/job/24281063026?pr=20770, i dont understand

**Comment by chyroc** [comment]
@baskaryan thank you, can you review this

File: libs/community/langchain_community/chat_models/__init__.py [hunk 1]
```diff
@@ -42,6 +42,9 @@
     from langchain_community.chat_models.cohere import (
         ChatCohere,  # noqa: F401
     )
+    from langchain_community.chat_models.coze import (
+        ChatCoze,  # noqa: F401
+    )
```
Comment (baskaryan) [inline libs/community/langchain_community/chat_models/__init__.py:47]: need to add in module lookup as well (starting L217)

**Comment by chyroc** [comment]
@baskaryan all fiexed, please review

**Comment by chyroc** [comment]
@baskaryan all fiexed, please review, thanks

**Pull request merged**